directory_header_footer/templates/directory_header_footer/header.html

```diff
@@ -1,66 +1,74 @@
 {% load static %}
 <header id="header" class="static-header-footer"><a id="skip-link" href="#content" class="verbose">Skip to main content</a>
+  {% if header_footer_language_select %}
+  <section role="dialog" data-component="language-selector-dialog" data-lang="en" data-label="UK" data-name="English" class="language-selector-dialog">
+    <p id="great-languages-selector">Go to our international site</p>
+    <p>The UK supports international trade and investment, and welcomes travellers and students from across the world.</p>
+    <div data-node="languages" class="languages">
+      <ul aria-labelledby="great-languages-selector" class="navigation">
+        <li><a href="{{ header_footer_urls.great_export_home }}/international?lang=en">English (International)</a></li>
+        <li><a href="{{ header_footer_urls.great_export_home }}/international?lang=zh">中国</a></li>
+        <li><a href="{{ header_footer_urls.great_export_home }}/international?lang=de">Deutsch</a></li>
+        <li><a href="{{ header_footer_urls.great_export_home }}/international?lang=ja">日本語</a></li>
+        <li><a href="{{ header_footer_urls.great_export_home }}/international?lang=es">Español</a></li>
+        <li><a href="{{ header_footer_urls.great_export_home }}/international?lang=pt">Português</a></li>
+        <li><a href="{{ header_footer_urls.great_export_home }}/international?lang=ar">عربى</a></li>
```

Comment:
the available languages are really settings, which are configurable. to use the languages defined in setting.py please use 
```
                {% for code, label in LANGUAGES %}
                    <li><a href="{{ header_footer_urls.great_export_home }}/international?lang={{ code }}">{{ label }}</a></li>
                {% endfor %}
```

note `LANGUAGES` is automatically exposed to the template when the `django.template.context_processors.i18n` context processor is in play.

for tests you'll need to add the following to conftest

```
LANGUAGES = [
    ('en-gb', 'English'),               # English
    ('zh-hans', '简体中文'),             # Simplified Chinese
    ('de', 'Deutsch'),                  # German
    ('ja', '日本語'),                    # Japanese
    ('es', 'Español'),                  # Spanish
    ('pt', 'Português'),                # Portuguese
    ('ar', 'العربيّة'),                 # Arabic
]
```

and for the test you would need to manaully expose the `LANGUAGE` context variabele to the template